src/scripts/fetch-routes.ts

```diff
@@ -89,6 +89,14 @@ const fetchDatoQuery = ({
     });
 };
 
+const fetchBlogPagesRoutes = async ({ locale } : { locale: string }) => {
+  const operation = "allBlogPosts";
+  const meta = await fetchMetaForOperation({ operation, locale });
+  const { count } = meta[`_${operation}Meta`];
+  const pages = Math.ceil(count / 20);
```

Comment:
Didn't knew were to place the 20 so that it can be used here and on the pagination